app/app/components/MemorableLines.module.css

```diff
@@ -0,0 +1,53 @@
+.memorableLines {
+    width: 100%;
+    padding: 40px 20px;
+    text-align: center;
+    margin: 50px 0 0 0;
+}
+
+.linesContainer {
+    display: flex;
+    justify-content: space-evenly;
+    flex-wrap: wrap;
+}
+
+.line {
+    max-width: 500px;
+    margin: 20px;
+    background-color: #ffffff;
```

Comment:
The CSS variables are especially useful for the colors. (Okay this one is easy to read)
I don't know what color is `#e344a1`, but I can easily understand if it says `var(--background-color)` or `var(--card-border-color)`